scripts/lua.sh

```diff
@@ -13,7 +13,17 @@ fi
 
 # LUA_T= and LUAC_T= disable building lua & luac
 # -Dgetlocaledecpoint()=('.') fixes bionic missing decimal_point in localeconv
-make CC="arm-linux-androideabi-gcc -Dgetlocaledecpoint\(\)=\(\'.\'\)" PLAT=linux LUA_T= LUAC_T= -j6
+if [ "$os" == "macosx" ]; then
+	if hash ginstall 2>/dev/null; then
+		sed -i.bak "s/INSTALL= install/INSTALL= ginstall/" Makefile
```

Comment:
We'd like to avoid patching the Makefile, can't you pass this on the command line?